python/py-configparser/Portfile

```diff
@@ -5,37 +5,47 @@ PortGroup           python 1.0
 
 name                py-configparser
 epoch               1
-version             4.0.2
+version             5.0.2
 revision            0
+
 categories-append   devel
 platforms           darwin
 license             MIT
 supported_archs     noarch
 
-python.versions     27
-
-maintainers         {stromnov @stromnov} openmaintainer
+homepage            https://github.com/jaraco/configparser
 
 description         Configparser from Python 3.8 to Python 2.6-3.7
 long_description    ${description}
 
-homepage            https://github.com/jaraco/configparser/
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+maintainers         {stromnov @stromnov} openmaintainer
 
-distname            ${python.rootname}-${version}
+python.versions     27 39
 
-checksums           rmd160  9b6a5b12490e72c47ce2b1d190a35687c2d2c7eb \
-                    sha256  c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df \
-                    size    72498
+if {${name} ne ${subport}} {
 
-# 4.0.2 is the last py27-compatible version
-livecheck.type      none
+    # 4.0.2 is the last py27-compatible version
+    if {${python.version} == 27} {
+        version             4.0.2
+
+        depends_lib-append  port:py${python.version}-backports
+
+        distname            ${python.rootname}-${version}
+
+        checksums           rmd160  9b6a5b12490e72c47ce2b1d190a35687c2d2c7eb \
+                            sha256  c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df \
+                            size    72498
+    } else {
+        depends_build-append port:py${python.version}-toml
```

Comment:
this indentation doesn't follow the modeline, other than that looks good to me

Reply:
OK, fixed